content/theme/minimal-bootstrap-blog.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "minimal-bootstrap-blog"
+github: https://github.com/andrewhwanpark/minimal-bootstrap-blog
+demo: https://andrewhwanpark.github.io/minimal-bootstrap-blog/ # Please make sure this links to a theme demo and not your personal/business site. The demo url must use https
+author: andrewhwanpark
+date: 2021-07-20 # Enter the date you submitted the theme YYYY-MM-DD
+ssg:
+  - Jekyll # at least 1 ssg is required
+cms:
+  - No CMS # at least 1 cms is required. If your theme uses markdown (no CMS) the cms should be "No CMS"
+css: # css is optional
+  - Bootstrap-dark-5
+archetype: # archetype is optional
+  - Blog
+  - Portfolio
+description: minimal-bootstrap-blog is a no frills, fully-responsive, hyper-minimalistic dark/light theme
+---
+
+# minimal-bootstrap-blog
+
+minimal-bootstrap-blog is a no frills, fully-responsive, hyper-minimalistic dark/light theme made with bootstrap-dark-5: https://github.com/vinorodrigues/bootstrap-dark-5. The theme is heavily inspired by Mark Otto's personal site: https://markdotto.com/
+
+Demo: https://andrewhwanpark.github.io/minimal-bootstrap-blog/
+
+My personal site that uses the same theme: https://andrewhwanpark.github.io/
```

Comment:
Please dont include links to your personal, commercial or other external sites

Reply:
will do thx